Karma/Controllers/AdminController.cs

```diff
@@ -342,5 +344,79 @@ private void hideReportedPost(int id)
             post.State = (int)Post.PostState.Hidden;
             post.IsVisible = false;
         }
+
+        [HttpGet]
+        //[ValidateAntiForgeryToken]
```

Comment:
[ValidateAntiForgeryToken] shouldn't be commented out (applies to all places where such thing is done).